exec_manager/job.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2021 - 2022 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""class for job"""
+
+from uuid import UUID
+
+from exec_manager.exec_profile import ExecProfile
+from exec_manager.job_status_type import JobStatusType
+
+
+class Job:
```

Comment:
Job sollte am Besten eine abstrakte Klasse sein:
```suggestion
from abc import ABC, abstractmethod
from uuid import UUID

from exec_manager.exec_profile import ExecProfile
from exec_manager.job_status_type import JobStatusType


class Job(ABC):
```